src/cli/node/json_rpc_client.rs

```diff
@@ -995,7 +996,7 @@ pub fn get_node_stats(addr: SocketAddr) -> Result<(), failure::Error> {
     let sync_status: SyncStatus = parse_response(&response)?;
 
     if let Some(current_epoch) = sync_status.current_epoch {
-        if sync_status.synchronized {
+        if sync_status.node_state.unwrap() == StateMachine::Synced {
```

Comment:
when the node_state is None?

Reply:
I guess that can only happen if the state is not initialized yet, so maybe we can have a heck for that with another if let Some(status) = status. That is another level of indentation but I guess it looks better than an unwrap. Or maybe at least we should do an unwrap_or_else, returning an error.

Reply:
Actually it is never none, since when the node is initialized the state is waitingConsensus. I've changed it so that the node_state is not an option anymore

Reply:
good

Reply:
breaking change then